<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>issue_book_ui</class>
 <widget class="QDialog" name="issue_book_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Issue Book</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#CBE6C7;</string>
  </property>
  <widget class="QPushButton" name="issue_book_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>375</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #122C4F;
	border: 1px solid #000000;
    color: #CBE6C7;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #0D203A;
}

QPushButton:pressed {
    background-color: #091525;
}
</string>
   </property>
   <property name="text">
    <string>Issue Book</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>300</y>
     <width>65</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;</string>
   </property>
   <property name="text">
    <string>Due Date :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>100</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>2</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;</string>
   </property>
   <property name="text">
    <string>Book Id : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>2</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_9">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>2</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="text">
    <string>ISSUE BOOK</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;</string>
   </property>
   <property name="text">
    <string>Member Id : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;</string>
   </property>
   <property name="text">
    <string>Issue Date : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>2</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="book_id_entry">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>185</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;
border-radius:10px;
border: 1px solid #122C4F;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>278</y>
     <width>100</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>288</y>
     <width>110</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>110</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="member_id_entry">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>115</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;
border-radius:10px;
border: 1px solid #122C4F;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="issue_date_edit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>255</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;
border-radius:10px;
border: 1px solid #122C4F;</string>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="due_date_edit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>325</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;
border-radius:10px;
border: 1px solid #122C4F;</string>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
